features/0592-indy-attachments/README.md

```diff
@@ -346,6 +365,11 @@ An example implementation in Python can be found [here](https://github.com/hyper
 
 A gist of test value pairs can be found [here](https://gist.github.com/swcurran/78e5a9e8d11236f003f6a6263c6619a6).
 
+#### Notes on Encoding Claims
+
+- In converting any string integer to an integer, leading 0s in the string are (by definition) not part of the integer. They leading 0's remain in the (untouched) `raw` value.
```

Comment:
```suggestion
- In converting any string integer to an integer, leading 0s in the string are (by definition) not part of the integer. The leading 0's remain in the (untouched) `raw` value.
```

Reply:
I'm not sure I fully understand, but does that mean the value of `012` will be encoded as `12`? If we have the creditcard CVC code it will be impossible for it to start with a `0`. IMO it should be encoded as as the string value (so `012`). Or is that what this suggests?

Reply:
The "raw" value will remain "012".  The encode value will be the integer 12, and will NOT be the hash of "12".

As long as you use the raw value, which you should, you are good.  And of course, you should make sure that the encoded  value is the expected integer 12.

Does that make sense?